app/src/main/java/com/cornellappdev/android/eatery/model/MealType.java

```diff
@@ -0,0 +1,27 @@
+package com.cornellappdev.android.eatery.model;
+
+public enum MealType {
+
```

Comment:
delete

Reply:
this one newline is enforced in Google's linting/style guide